src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,8 @@
+.cards {
+  width: $page-width;
```

Comment:
It is said in the homework that page width is always >= 1024px, but  I don't think it means the card width should be the same. You can create a variable that will count the cards' max-width. (multiply card width by 4 + gaps between the cards), so there always will be 4 cards in a row. And you won't need to use 'padding: 0 40px;' here ;